.github/renovate.json5

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [ // See what config:recommended does: https://docs.renovatebot.com/presets-config/#configrecommended
+    "config:recommended",
+    // Dependency Dashboards are Issues Renovate creates which list all pending updates. We disable it
+    // because we don't need it, we can see the pending updates in the PRs. This is also noisy.
+    // https://docs.renovatebot.com/configuration-options/#dependencydashboard
+    // Enabled for now to see how it works and if it's useful, we can disable it later if needed.
+    // ":disableDependencyDashboard"
```

Comment:
```suggestion
```
I find the DependencyDashboard very useful.

If in the future there are dependencies that Renovate cannot parse, or Renovate detects deprecated dependencies, or finds that the configuration needs migration, you can always check these through the DependencyDashboard.

It also helps other contributors understand the status of the repository.

Disabling it is strongly discouraged.